# Generated by Django 4.2.5 on 2023-09-30 13:50

from django.db import migrations, models


class Migration(migrations.Migration):
    dependencies = [
        ("backend", "0003_rename_errors_error_and_more"),
    ]

    operations = [
        migrations.RemoveField(
            model_name="client",
            name="name",
        ),
        migrations.AddField(
            model_name="client",
            name="address",
            field=models.CharField(blank=True, max_length=100, null=True),
        ),
        migrations.AddField(
            model_name="client",
            name="city",
            field=models.CharField(blank=True, max_length=100, null=True),
        ),
        migrations.AddField(
            model_name="client",
            name="country",
            field=models.CharField(blank=True, max_length=100, null=True),
        ),
        migrations.AddField(
            model_name="client",
            name="first_name",
            field=models.CharField(default=None, max_length=32),
            preserve_default=False,
        ),
        migrations.AddField(
            model_name="client",
            name="last_name",
            field=models.CharField(default=None, max_length=32),
            preserve_default=False,
        ),
        migrations.AddField(
            model_name="client",
            name="phone_number",
            field=models.CharField(blank=True, max_length=100, null=True),
        ),
        migrations.AlterField(
            model_name="client",
            name="email",
            field=models.EmailField(blank=True, max_length=254, null=True),
        ),
    ]
